flexget/plugins/input/rss.py

```diff
@@ -184,6 +185,25 @@ def process_invalid_content(self, task, data, url):
             f.write(data)
         log.critical('I have saved the invalid content to %s for you to view', filepath)
 
+    def escape_content(self, content):
+        valid_escapes = (b'&quot;', b'&apos;', b'&lt;', b'&gt;', b'&amp;')
+        future_result = []
+        in_cdata_block = False
+
+        for idx, char in enumerate(content):
+
+            if isinstance(char, int):
+                char = tobytes(chr(char))
```

Comment:
After some investigating, the integers happen when iterating bytes in python 3. I think this way to convert to a length one bytestring is cleaner though:
`char = bytes([char])`

Reply:
I'm actually thinking remove the isinstance check, as the above will work fine on python2 as well.

Reply:
`char = bytes([char])` doesn't work  in python2:

```
  File "/home/finkel/work_self/Flexget/flexget/plugin.py", line 118, in wrapped_func
    return func(*args, **kwargs)
  File "/home/finkel/work_self/Flexget/flexget/plugins/input/rss.py", line 311, in on_task_input
    content = self.escape_content(content)
  File "/home/finkel/work_self/Flexget/flexget/plugins/input/rss.py", line 195, in escape_content
    char = bytes([char])
  File "/usr/lib/python2.7/site-packages/future/types/newbytes.py", line 115, in __new__
    raise ValueError('bytes must be in range(0, 256)')
```